src/shared/watchdog.c

```diff
@@ -114,9 +117,31 @@ int watchdog_set_timeout(usec_t *usec) {
         return r;
 }
 
+usec_t watchdog_runtime_wait(void)
+{
```

Comment:
coding style: we place `{`on the same line of the function name, see:

https://systemd.io/CODING_STYLE